includes/class-wc-stripe-helper.php

```diff
@@ -516,183 +476,33 @@ public static function get_legacy_payment_method( $id ) {
 	 * The ids are mapped to the corresponding equivalent UPE method ids for rendeing on the frontend.
 	 *
 	 * @return array
+	 *
+	 * @deprecated 10.1.0 This method will be removed in future versions.
```

Comment:
Nit: Another version fix.
```suggestion
	 * @deprecated 10.2.0 This method will be removed in future versions.
```